### DISCUSSION TITLE
Primereact Atlantis theme horizontal menu not working

### DISCUSSION THREAD

**Discussion by rahulAnand1999** [primefaces/community#2327] [PrimeReact] [answered]
In the primereact atlantis theme the horizontal menu is not working .
The menu gets shifted to the top but on clicking of it , it doesnot open the child menus.

I have purchased it recently. Please provide the solution asap.

**Comment by cagataycivici** [comment]
Thank you, I've informed the team.

**Comment by nitrogenous** [comment]
Hello there @rahulAnand1999 

We have addressed the issue you mentioned and are pleased to inform you that it has been fixed. We will be releasing a new version as soon as possible.

Thank you for your contribution and support to PrimeTek.

Warmest regards,
PrimeReact Team

**Comment by rahulAnand1999** [comment]
Hi Primefaces/Community Support Team,

I can see the horizontal menu changes in the atlantis website, but as i
have purchased it already will i need to pay for the update also?
I can't see any changes in my account but it says extended update for $39.

Please help asap.

**Comment by nitrogenous** [comment, marked as answer]
Hi @rahulAnand1999 ,

You don't need to purchase an extended update service at this moment. We will be releasing a new version with a bug fix this week.

In the meantime, you can fix the bug yourself by changing the code in **layout/AppMenuItem.tsx** at line 139 from:

`
'active-menuitem': active && !props.root
`

to:

`
'active-menuitem': active
`

Thank you for your understanding and patience
